docs/src/prototypes/chatMessages/ThreadedMessages/ThreadReplies.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import {
+  AvatarProps,
+  Avatar,
+  ChatMessageProps,
+  Button,
+  ChatItem,
+  ChatItemProps,
+  Chat,
+} from '@stardust-ui/react'
+import repliesButtonBehavior from './repliesButtonBehavior'
+import ScreenReaderHeaderText from './SreenReaderHeaderText'
+
+export type ThreadReplyProps = ChatMessageProps & {
+  avatar?: AvatarProps
+}
+
+type ThreadRepliesProps = {
+  replies?: ThreadReplyProps[]
+}
+
+const ThreadReplies = (props: ThreadRepliesProps) => {
+  const [expanded, setExpanded] = React.useState(false)
+
+  const renderCollapseAllOrRepliesButton = () => {
+    const { replies } = props
+    if (!replies) {
+      return null
+    }
+
+    const repliesCount = replies.length
+
+    if (repliesCount === 0 || repliesCount === 1) {
+      return null
+    }
+
+    const authorName1 = replies[0].author
+    const authorName2 = replies[1].author
+    const remainReplies = repliesCount - 2
+    const moreRepliesLeft = remainReplies > 0
+    const buttonText = expanded
+      ? 'Collapse all'
+      : `${repliesCount} replies from ${authorName1} and ${authorName2}${
+          moreRepliesLeft ? `, and ${remainReplies} others` : ''
+        }`
+
+    return (
+      <Button
+        as="a"
+        className="ui-button__reply"
+        fluid
+        accessibility={repliesButtonBehavior}
+        onClick={() => setExpanded(!expanded)}
+      >
+        <ScreenReaderHeaderText level="5" text={buttonText} />
+        <div aria-hidden="true">{buttonText}</div>
```

Comment:
Should we use `screenReaderContainerStyles` there?

Reply:
they should be applied to `ScreenReaderHeaderText` only - meaning that it will be "visible" for SR but hidden from user's eyes.
`aria-hidden` does vice versa - hides from SR, but remains visible